examples/discovery-search-app/cypress/fixtures/spelling/correctedQuery.json

```diff
@@ -0,0 +1,1683 @@
+{
+  "matching_results": 17,
+  "retrieval_details": {
+    "document_retrieval_strategy": "untrained"
+  },
+  "suggested_refinements": [
+    {
+      "text": "top of IBM"
+    },
+    {
+      "text": "AI"
+    },
+    {
+      "text": "ons"
+    },
+    {
+      "text": "Installing"
+    },
+    {
+      "text": "OS Connector"
+    },
+    {
+      "text": "IBM Db2"
+    },
+    {
+      "text": "Assistant"
+    },
+    {
+      "text": "discovery"
+    },
+    {
+      "text": "add"
+    },
+    {
+      "text": "Parent topic"
+    }
+  ],
+  "aggregations": [
+    {
+      "type": "term",
+      "field": "enriched_text.entities.text",
+      "name": "entities",
+      "results": [
+        {
+          "key": "IBM",
+          "matching_results": 17
+        },
+        {
+          "key": "Watson",
+          "matching_results": 17
+        },
+        {
+          "key": "ibm",
+          "matching_results": 14
+        },
+        {
+          "key": "Kubernetes",
+          "matching_results": 9
+        },
+        {
+          "key": "Pak",
+          "matching_results": 8
+        },
+        {
+          "key": "2.1.0.1",
+          "matching_results": 7
+        },
+        {
+          "key": "Red Hat",
+          "matching_results": 6
+        },
+        {
+          "key": "100 GB",
+          "matching_results": 5
+        },
+        {
+          "key": "@MASTER_1_IP",
+          "matching_results": 5
+        },
+        {
+          "key": "MongoDB",
+          "matching_results": 5
+        }
+      ]
+    }
+  ],
+  "results": [
+    {
+      "document_id": "e69fd25f-5cbf-4f53-bef3-67834ac1965b",
+      "result_metadata": {
+        "collection_id": "fbafd5e1-9cf1-c24c-0000-016eeb0a5c9f",
+        "document_retrieval_source": "search",
+        "confidence": 0.0598
+      },
+      "enriched_text": [
+        {
+          "entities": [
```

Comment:
i wonder how many of these entities are relevant for the correctedQuery json

Reply:
I stripped this file down a bit to make it more manageable